pkg/api/v1alpha1/kepler_types.go

```diff
@@ -95,6 +95,8 @@ type ExporterSpec struct {
 	// TODO: fix the default version before dev-preview
 
 	// +kubebuilder:default=9103
+	// +kubebuilder:validation:Maximum=65535
+	// +kubebuilder:validation:Minimum=1
```

Comment:
a user created port cannot take values as low as 1

Reply:
why not? should it be > 1024?

Reply:
I have validated, and it works, could you please check if that works for you? Try setting the port to 1 and

```
 kop/wt/e2e-test kubectl port-forward services/kepler-exporter-svc 3000:1 --address=0.0.0.0 -n openshift-kepler-operator
```
Forwarding from 0.0.0.0:3000 -> 1

```
curl http://localhost:3000/metrics | ag 'kepler_'
```

Reply:
Whatever may be case be, this should be reflected in the `Availablility` status. For the record, I am open to increasing the min value. But do note that for k8s service, the allowed range starts from `1`

Reply:
i think so, creating a lower valued port needs SELinux permissions